component-overview/examples/form/InputGroup-onColoredBg.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react';
+import { Datepicker } from '@sb1/ffe-datepicker-react';
+import { Dropdown } from '@sb1/ffe-dropdown-react';
+import { InputGroup, Checkbox, Input, TextArea } from '@sb1/ffe-form-react';
+import { SecondaryButton } from '@sb1/ffe-buttons-react';
+
+() => {
+    const [showErrors, setShowErrors] = useState(false);
+
+    return (
+        <div style={{ background: '#002776', padding: 16 }}>
```

Comment:
Er det en måte man kunne brukt f.eks css-variabler her i stedet? Går det ikke så går det ikke, men selv om det bare er i dokumentasjonen så er hardkodede farger helt forferdelig å jobbe med

Reply:
Dette er ju html og ikke css så det lar seg nok ikke gjøres direkte her. I så fall måtte man laget klasser for det, men vet ikke om det gir mening før dom som leser eksemplet?

Reply:
eller importer javascript varianterna av fargekonstanterna. Avtand har ikke noen konstanter mener jag

Reply:
Kjører på med `import { blueRoyal } from '@sb1/ffe-core';`

Reply:
`blueRoyal` hører til det gamle fargeskjemaet og bør helst ikke brukes. Fargen er egentlig kun fortsatt tilgjengelig i og med at den er deprecated og ikke slettet. Det samme gjelder alle farger som _ikke_ starter på `@ffe-farge` i [colors.less](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-core/less/colors.less).